src/main/java/frc/robot/subsystems/tower/ArmImpl.java

```diff
@@ -25,13 +28,39 @@
         POSITION_TICKS = UnmodifiableBidiMap.unmodifiableBidiMap(positionTicks);
     }
 
-    // TODO set up TalonSRX for Position PID:
-    // https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages/blob/master/Java/PositionClosedLoop/src/main/java/frc/robot/Robot.java
     private final WPI_TalonSRX controller;
     private Position targetPosition = Position.STARTING_CONFIG;
 
+    //Arm PID
```

Comment:
Do all of these really need to be added to the class as fields? They're only ever referenced within the constructor for configuring the PID controller, and are/will be never needed again for the lifetime of the ArmImpl instance.

Reply:
You're right, I'll fix that